src/structures/RichEmbed.js

```diff
@@ -128,21 +128,23 @@ class RichEmbed {
    * @param {Date} [timestamp=current date] The timestamp
    * @returns {RichEmbed} This embed
    */
-  addTimestamp(timestamp = new Date()) {
+  setTimestamp(timestamp = new Date()) {
     this.timestamp = timestamp;
     return this;
   }
 
   /**
    * Adds a field to the embed (max 25)
    * @param {string} name The name of the field
-   * @param {string} value The value of the field
+   * @param {StringResolvable} value The value of the field
    * @param {boolean} [inline=false] Set the field to display inline
    * @returns {RichEmbed} This embed
    */
-  addField(name, value, inline = false) {
+  addField(name, value = '\u200b', inline = false) {
```

Comment:
instead of doing this, change the docstring to say that value should not be left undefined/an empty string, or add a check.